tutorials/resources/yolov8/yolov8n.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
Same comment about license. if you didn't change the file at all, maybe download it online